Lib/unittest/case.py

```diff
@@ -1206,10 +1206,12 @@ def assertDictContainsSubset(self, subset, dictionary, msg=None):
 
 
     def assertCountEqual(self, first, second, msg=None):
-        """An unordered sequence comparison asserting that the same elements,
-        regardless of order.  If the same element occurs more than once,
+        """An unordered sequence comparison asserting that the same elements
+        are present, regardless of order. If the same element occurs more than once,
         it verifies that the elements occur the same number of times.
```

Comment:
Just a comment from a no-English-speaker:  Why don't write

```suggestion
         it verifies that this elements occur the same number of times.
```
Because, I think that you are talking about the "elements", so I think that would be better reference to that elements like `this elements`

Reply:
`the` is ok with `elements` and because it's plural, maybe the correct article is `these` or `those`

Reply:
ohh! yes! `those`.